packages/react-conformance/src/defaultTests.tsx

```diff
@@ -63,7 +63,7 @@ export const defaultTests: TestObject = {
         displayName = displayName.replace(/^Styled/, '');
       }
 
-      expect(displayName).toEqual(testInfo.displayName);
+      expect(displayName).toMatch(new RegExp(`^(Customized|Styled)?${testInfo.displayName}(Base)?$`));
```

Comment:
instead of changing this. I think we can just replace 
```
const constructorName = Component.prototype?.constructor.name;
if (constructorName === 'Wrapped') {
    displayName = displayName.replace(/^Styled/, '');
}
```
with 
```
displayName = displayName.replace(/^Styled/, '');
```
?

cc @sopranopillow

Reply:
It’s functionally the same either way. I feel it’s cleaner with the RegExp.

If you feel strongly enough, I can change it. Otherwise, I’ll merge tomorrow.

Reply:
I don't mind to use regex. if so, we should remove the logic above. not need to have both
```
      const constructorName = Component.prototype?.constructor.name;
      let displayName = Component.displayName || constructorName;

      // This check is needed in case the Component is wrapped with the v7 styled() helper, which returns a wrapper
      // component with constructor name Wrapped, and adds a Styled prefix to the displayName. Components passed to
      // styled() typically have Base in their name, so remove that too.
      displayName = displayName.replace(/Base$/, '');
      displayName = displayName.replace(/^Customized/, '');
      if (constructorName === 'Wrapped') {
        displayName = displayName.replace(/^Styled/, '');
      }
```

Reply:
Whoops! Fixed. 😝